### DISCUSSION TITLE
Data List: Set value programmatically

### DISCUSSION THREAD

**Discussion by jpabdilla-gfi** [leekelleher/umbraco-contentment#129] [Question] [answered]
Is it possible to set the values programmatically of contentment data list where data source is Umbraco content and list editor is a checkbox list?

For example using Umbraco's checkbox list this can be achieved as follows: https://our.umbraco.com/Documentation/Fundamentals/Backoffice/property-editors/built-in-property-editors/CheckBox-List/#add-values-programmatically

**Comment by leekelleher** [comment, marked as answer]
@jpabdilla-gfi Yes, it's possible.

The approach would be similar to the example in the Umbraco documentation. Except the values would need to be the UDI of the content items, e.g.

    content.SetValue("superHeros", JsonConvert.SerializeObject(new[] { "umb://document/4fed18d8c5e34d5e88cfff3a5b457bf2", "umb://document/4fed18d8c5e34d5e88cfff3a5b457bf2" }));

Hope this helps?

**Reply by jpabdilla-gfi** [reply]
@leekelleher Works. Thanks.
